modules/machineset-gcp-enabling-gpu-support.adoc

```diff
@@ -0,0 +1,130 @@
+--
+
+You can define which supported GPU to use for an instance by using the Machine API.
+
+You can configure machines in the N1 machine series to deploy with a GPU attached, or to use a guest accelerator.
+
+Machines in the A2 machine series come with associated GPUs, and cannot use guest accelerators.
+
+[NOTE]
+====
+GPUs for graphics workloads are not supported.
+====
+
+.Procedure
+
+. In a text editor, open the YAML file for an existing machine set or create a new one.
+
+. Specify a GPU configuration under the `providerSpec` field in your machine set YAML file. See the following examples of valid configurations:
++
+.Example configuration for the A2 machine series:
+[source,yaml]
+----
+  providerSpec:
+    value:
+      machineType: a2-highgpu-1g <1>
+      onHostMaintenance: Terminate <2>
+      restartPolicy: Always <3>
+----
+<1> Specify the machine type. Ensure that the machine type is included in the the A2 machine series.
+<2> When using GPU support, you must set `onHostMaintenance` to `Terminate`.
+<3> Specify the restart policy for machines deployed by the machine set. Allowed values are `Always` or `Never`.
++
+.Example configuration for the N1 machine series:
+[source,yaml]
+----
+providerSpec:
+  value:
+    gpus:
+    - count: 1 <1>
+      type: nvidia-tesla-p100 <2>
+    machineType: n1-standard-1 <3>
+    onHostMaintenance: Terminate <4>
+    restartPolicy: Always <5>
+----
+<1> Specify the number of GPUs to attach to the machine.
+<2> Specify the type of GPUs to attach to the machine. Ensure that the machine type and GPU type are compatible.
+<3> Specify the machine type. Ensure that the machine type and GPU type are compatible.
+<4> When using GPU support, you must set `onHostMaintenance` to `Terminate`.
+<5> Specify the restart policy for machines deployed by the machine set. Allowed values are `Always` or `Never`.
++
+.Example configuration for attaching guest accelerators:
+[source,yaml]
+----
+providerSpec:
+  value:
+    guestAccelerators:
+     - acceleratorCount: 1 <1>
+       acceleratorType: nvidia-tesla-p100 <2>
+    machineType: n1-standard-1 <3>
+    onHostMaintenance: Terminate <4>
+    restartPolicy: Always <5>
+----
+<1> Specify the number of GPUs to attach to the machine.
+<2> Specify the type of GPUs to attach to the machine. Ensure that the machine type and GPU type are compatible.
+<3> Specify the machine type. Ensure that the machine type and GPU type are compatible.
+<4> When using GPU support, you must set `onHostMaintenance` to `Terminate`.
+<5> Specify the restart policy for machines deployed by the machine set. Allowed values are `Always` or `Never`.
```

Comment:
@jeana-redhat  looks like this part is not needed and we need to delete it. Others looks good to me, thank you!

Reply:
Yes this whole part should not be here. `guestAccelerators` was an old name for the current `GPUs` field.